api/src/main/java/com/codeforcommunity/dto/team/AddGoalRequest.java

```diff
@@ -0,0 +1,74 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.security.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddGoalRequest extends ApiDto {
+  private Integer goal;
+  private Integer teamId;
+  private Timestamp startAt;
+  private Timestamp completeBy;
+
+  public AddGoalRequest(Integer goal, Integer teamId, Timestamp startAt, Timestamp completeBy) {
+    this.goal = goal;
+    this.teamId = teamId;
+    this.startAt = startAt;
+    this.completeBy = completeBy;
+  }
+
+  private AddGoalRequest() {}
+
+  @Override
+  public List<String> validateFields(String fieldPrefix) throws HandledException {
```

Comment:
This is super nitpicky but let's be consistent about where in the DTO class this method goes. I think by convention we've been doing it as the last thing in the class